assets/js/alerting.js

```diff
@@ -0,0 +1,118 @@
+// assets/js/alerting.js
```

Comment:
This module should be committed to master. When when we pull master into emetrotel, we'll have it in emetrotel.